html_css/intermediate_css/custom-properties.md

```diff
@@ -136,6 +136,7 @@ Using the `prefers-color-scheme` media query can be pretty helpful for users sin
 1. Read through MDN's [Using CSS custom properties](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties) page starting at the "Inheritance of custom properties" section.
 2. Open the inspector on this page to inspect the styles and see how Odin uses some custom properties.
 </div>
+3. [This video](https://www.youtube.com/watch?v=PHO6TBq_auI) is a great introduction to CSS custom properties. Go ahead and watch it.
```

Comment:
I like the video, although I believe it doesn't cover anything new that isn't already covered in the lesson and the assignment itself. Hence, the video isn't necessary to complete the lesson. Nevertheless, we could add it to the Additional Resources section of the lesson. What do you think about that?

Reply:
I think that works out just fine. Thank you.

Reply:
Could you make that change please?